uber/templates/signups/shifts.html

```diff
@@ -1,22 +1,202 @@
-<!doctype html>
-<html>
-<head>
-    <title>{{ name }}'s shifts</title>
-    <link rel="stylesheet" type="text/css" href="../static/styles/styles.css" />
+{% extends 'signup_base.html' %}
+
+{% block page_style %}
+<style>
+.fc-toolbar.fc-header-toolbar {
+	margin-left:15px;
+	margin-right:15px;
+}
+.shift_icon {
+	border-radius:15px;
+	width:30px;
+	height:30px;
+	display:inline-block;
+}
+.shift_taken {
+	background-color:#239875;
+}
+.shift_available {
+	background-color:#305fc9;
+}
+.fc-list-item-title .shift_button {
+	color:white;
+	float:right;
+}
+</style>
+{% endblock %}
+{% block main_content %}
+<div class="hidden csrf_token">
+{% csrf_token %}
+</div>
+
+
+<div class="row">
+	<div class="col-md-12 text-center">
+	<span style="font-size:16pt ; font-weight:bold" class="ng-binding">T U's available shifts</span>
+   	</div>
+</div>
+<div class="row">
+	<div class="col-md-12 text-center">
+   		<span style="font-size:12pt ; font-style:italic" class="ng-binding">(If you are not T U, <a href="login">log in here</a>.)</span>
+	</div>
+</div>
+<div class=row">
+	<div class="col-md-12 text-center">
+		<table width="100%" style="text-align:center">
+   		<tbody><tr>
+       			<td ng-bind-html-unsafe="'../static_views/stafferComps.html' | popupLink:'See What You Get For Volunteering'"></td>
+       <!-- ngIf: !c.HIDE_SCHEDULE -->
+   </tr>
+   </tbody></table>
+   <br>
+   {% if 'schedule' in c.QUERY_STRING %}
```

Comment:
I can see that you did this to try to emulate some of the functionality of the angular app you're replacing, and this is a fine way to do it, but there may be a more user-friendly way. For example, what if there was just a "Show/Hide Available Shifts" button you could click to update the calendar? If we can do that without a page reload, that'll be best for volunteers.